otx/algorithms/action/adapters/mmaction/models/heads/movinet_head.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+import torch.nn as nn
+from mmaction.models.builder import HEADS
+from mmaction.models.heads.base import BaseHead
+from mmcv.cnn import normal_init
+
+from ..backbones.movinet import ConvBlock3D
```

Comment:
Does this have to be imported? If so, it doesn't seem appropriate for the class (`ConvBlock3D`) to be placed in the `backbone.movinet` file.